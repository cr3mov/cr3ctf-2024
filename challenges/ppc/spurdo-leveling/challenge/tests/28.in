257577619327902396 905581440617821551
151494 76571016155177
151494 8478339199900
