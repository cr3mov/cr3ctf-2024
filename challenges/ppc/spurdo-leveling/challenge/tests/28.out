740796700 7620 7620
